gulpfile.js

```diff
@@ -433,9 +428,9 @@
 			}
 		},
 
-		logToTerminal: () => {
-			logToTerminal.apply(this, arguments);
-		},
+		logToTerminal: () => logToTerminal.apply(this, arguments),
```

Comment:
@erik-ciq Sorry I just assumed the problem was the same as envOrArg without testing, but it's not. arguments is a reserved word with a lot of junk in it, but I don't think our message is inside. It needs to pass in args like envOrArg.
logToTerminal: (...args) => logToTerminal.apply(this, args),

Reply:
That makes sense. Updated to use args instead of arguments.